Makefile

```diff
@@ -19,7 +19,7 @@ PACKAGE_NAME?=github.com/projectcalico/calico
 
 # Determine whether there's a local yaml installed or use dockerized version.
 # Note in order to install local (faster) yaml: "go get github.com/mikefarah/yq.v2"
-YAML_CMD:=$(shell which yq.v2 || echo docker run --rm -i calico/yaml)
+YAML_CMD:=$(shell which yq.v2 || echo sudo docker run --rm -i calico/yaml)
```

Comment:
I don't like the idea of running commands as sudo in our makefiles - the user running this should have permissions to run docker commands without sudo. We should just add that as a requirement.

Reply:
ok., thats fair